agent/tool-scripts/meta.json

```diff
@@ -0,0 +1,48 @@
+{
+	"transient":{
+		"blktrace": null,
```

Comment:
What does a null represent here?

Reply:
Once we want to add additional tool info/properties, those nulls will be replaced. For now null simply represents no additional data